src/daemon/ca/manager.rs

```diff
@@ -544,24 +552,38 @@ impl CaManager {
 
         let content = ca.verify_rfc6492(msg)?;
 
-        let (child, recipient, content) = content.unpack();
+        let (child_handle, recipient, content) = content.unpack();
+
+        // If the child was suspended, because it was inactive, then we can now conclude
```

Comment:
Could there ever be a situation where a child was suspended for a reason other than inactivity and then that it might not be right to unsuspend it based solely on fresh activity?